packages/ember-data/tests/integration/serializers/embedded-records-mixin-test.js

```diff
@@ -1157,6 +1157,46 @@ test("serialize with embedded object (belongsTo relationship)", function() {
   });
 });
 
+test("serialize with embedded object (polymorphic belongsTo relationship)", function() {
+  env.registry.register('serializer:super-villain', DS.RESTSerializer.extend(DS.EmbeddedRecordsMixin, {
+    attrs: {
+      secretLab: { embedded: 'always' }
+    }
+  }));
+
+  SuperVillain.reopen({
+    secretLab: DS.belongsTo('secret-lab', { polymorphic: true })
+  });
+
+  var json, tom;
+  run(function() {
+    tom = env.store.createRecord(
+      'super-villain',
+      { firstName: "Tom", lastName: "Dale", id: "1",
```

Comment:
can we clean up the formatting here a bit? it'd be preferable for this to span multiple lines:

```js
{
  firstName: "tom",
  lastName: "Dale",
  id: "1",
}
```